src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -19624,5 +19792,56 @@ class C
                                         ILiteralOperation (OperationKind.Literal, Type: System.Char, Constant: a, IsInvalid) (Syntax: ''a'')
                 """);
         }
+
+        [Theory]
+        [InlineData("""$"{s}{s}{s}{s}{s}" """)]
+        [InlineData("""$"{s}" + $"{s}" + $"{s}" + $"{s}" + $"{s}" """)]
+        public void InterpolatedStringWith5StringParts_NoInlineArraySupport(string expression)
+        {
+            var code = $"""
+                string s = "a";
+                System.Console.WriteLine({expression});
+                """;
+
+            var comp = CreateCompilation(code, targetFramework: TargetFramework.Net100);
```

Comment:
>targetFramework: TargetFramework.Net100 [](http://example.com/codeflow?start=47&length=39)

Can this test use default TargetFramework?

Reply:
No. I'm intentionally testing when `params ReadOnlySpan<string>` exists, but inline arrays are not supported by the runtime.